tests/test_sandbox/test_ecs_fargate.py

```
@@ -439,39 +442,72 @@ async def test_exec_delegates_to_client(self):
+
+        sb, client = self._started_sb()
+
+        async def _slow(*args, **kwargs):
+            await _asyncio.sleep(0.1)
+            return ExecResult("", "", 0)
+
+        client.exec.side_effect = _slow
+        t0 = _time.monotonic()
+        await _asyncio.gather(*(sb.exec(f"echo {i}") for i in range(20)))
+        elapsed = _time.monotonic() - t0
+        assert elapsed < 1.0, f"20 × 0.1s calls took {elapsed:.2f}s — serialized?"
```

Ruff (0.15.14)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)